src/content/blog/watch-ddev-local-from-scratch-with-windows-wsl2.md

```diff
@@ -1,47 +1,107 @@
+
+## What You'll Learn
+
+**DDEV Fundamentals**: Understanding what DDEV is and why it's become the go-to solution for local web development. The video explains that DDEV is a Docker-based local development environment aimed at web developers, mostly PHP and Node developers. Key benefits include:
+
+- Developers can run websites on their local computer with almost no configuration
+- Multiple projects can run simultaneously, each with different configurations
+- Docker handles all the heavy lifting, so you don't even need PHP or Composer or Node on your host computer
+- First-class support across macOS, Linux, Windows, and WSL2 - it works the same on every platform
+
+**WSL2 Setup and Benefits**: WSL2 ([Windows Subsystem for Linux version 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-index)) transforms Windows development by providing a real Linux kernel running alongside Windows. The video covers the key advantages and considerations:
+
+_Advantages:_
+
+- Complete Linux environment on your Windows machine - fast and capable
+- Amazingly fast web serving with DDEV/Docker CE
+- You're working with an environment much like the one you'll deploy to
+
+_Considerations:_
+
+- Linux on your Windows machine means yet another system to learn and remember
+- Context switches between Windows and WSL2 environments
+- You must work in the WSL2 filesystem rather than Windows filesystem (optimized for web apps, not Microsoft Word)
+
+**Complete Installation Process**: The video covers [every step](/get-started) of the installation process, including:
+
+- Setting up WSL2 from scratch using the `wsl --install` command
+- Running the new DDEV Windows installer that automatically configures your distro for Docker CE
+- Understanding why you need to work in the WSL2 file system instead of the Windows NTFS filesystem for optimal performance
+
+**Real-World Usage**: Beyond just installation, you'll see DDEV in action with:
+
+- Creating a simple PHP project and launching it with trusted HTTPS certificates
+- Installing Drupal 11 using `ddev composer create-project`
+- Essential DDEV commands like `ddev describe`, `ddev snapshot`, and `ddev export-db`
+- Advanced IDE integration with both PhpStorm and VS Code, including Xdebug debugging
+
+**Professional Development Features**: The video demonstrates advanced workflows including:
+
+- Setting up Xdebug debugging in both PhpStorm and VS Code
+- Working with DDEV projects from within WSL2-integrated IDEs
+- Understanding Docker Desktop vs Docker CE trade-offs for professional development
+- Best practices for file system performance and cross-platform compatibility
+
+This screencast follows the official DDEV [WSL2 Installation Docs](https://ddev.readthedocs.io/en/stable/#installation-or-upgrade-windows-wsl2), but provides additional context, troubleshooting tips, and real-world examples to ensure your success.
+
+**Here's the video table of contents (opens on YouTube):**
```

Comment:
I added timestamps on the video itself here https://studio.youtube.com/video/1dr_4gPtFlQ/edit

And I wanted to add a link back to the blog, but it needs one-time verification

<img width="663" height="463" alt="image" src="https://github.com/user-attachments/assets/8f4cab33-7664-43d1-976d-0e9129134386" />

I can't do that because I can't login to your YouTube because it's protected by passkey, not 2FA.